pocketbase/docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pocketbase_app_1
+      APP_PORT: 8090
+      PROXY_AUTH_ADD: "false"
+  
+  app:
+    image: ghcr.io/muchobien/pocketbase:0.22.11@sha256:04dd9850554a24f82cfee060ff394179baaecc0f97e7721d9dd06818d26fd671
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/app/data:/pb_data
+      - ${APP_DATA_DIR}/app/public:/pb_public
+      - ${APP_DATA_DIR}/app/hooks:/pb_hooks
```

```suggestion
      - ${APP_DATA_DIR}/data/data:/pb_data
      - ${APP_DATA_DIR}/data/public:/pb_public
      - ${APP_DATA_DIR}/data/hooks:/pb_hooks
```

App data is usually stored in ${APP_DATA_DIR}/data